Week-2/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
@soumya213 looking good. One small issue

<img width="392" alt="Screenshot 2022-08-02 at 11 26 48 PM" src="https://user-images.githubusercontent.com/9133593/182442215-d6a6a8c6-8a5c-47b1-a9de-dca7e1b149cd.png">

Can you fix this issue alone ? it seems not aligned, because of footer it pushes the content away.